sormas-api/src/main/java/de/symeda/sormas/api/labmessage/LabMessageDto.java

```diff
@@ -0,0 +1,267 @@
+package de.symeda.sormas.api.labmessage;
+
+import java.util.Date;
+
+import de.symeda.sormas.api.Disease;
+import de.symeda.sormas.api.EntityDto;
+import de.symeda.sormas.api.person.Sex;
+import de.symeda.sormas.api.sample.PathogenTestResultType;
+import de.symeda.sormas.api.sample.PathogenTestType;
+import de.symeda.sormas.api.sample.SampleMaterial;
+import de.symeda.sormas.api.sample.SpecimenCondition;
+import de.symeda.sormas.api.utils.DataHelper;
+
+public class LabMessageDto extends EntityDto {
+
+	public static final String I18N_PREFIX = "Location";
+
+	public static final String SAMPLE_DATE_TIME = "sampleDateTime";
+	public static final String SAMPLE_RECEIVED_DATE = "sampleReceivedDate";
+	public static final String LAB_SAMPLE_ID = "labSampleId";
+	public static final String SAMPLE_MATERIAL = "sampleMaterial";
+	public static final String LAB_NAME = "LabName";
+	public static final String LAB_EXTERNAL_ID = "LabExternalId";
+	public static final String LAB_POSTAL_CODE = "LabPostalCode";
+	public static final String LAB_CITY = "LabCity";
+	public static final String SPECIMEN_CONDITION = "specimenCondition";
+	public static final String TEST_TYPE = "testType";
+	public static final String TESTED_DISEASE = "testedDisease";
+	public static final String TEST_DATE_TIME = "testDateTime";
+	public static final String RESULT = "result";
+	public static final String FIRST_NAME = "firstName";
+	public static final String LAST_NAME = "lastName";
+	public static final String SEX = "sex";
+	public static final String BIRTH_DATE_DD = "birthDateDD";
+	public static final String BIRTH_DATE_MM = "birthDateMM";
+	public static final String BIRTH_DATE_YYYY = "birthDateYYYY";
+	public static final String PERSON_POSTAL_CODE = "personPostalCode";
+	public static final String PERSON_CITY = "personCity";
+	public static final String PERSON_STREET = "personStreet";
+	public static final String PERSON_HOUSE_NUMBER = "personHouseNumber";
+	public static final String LAB_MESSAGE_DETAILS = "labMessageDetails";
+
+	private Date sampleDateTime;
+	private Date sampleReceivedDate;
+	private String labSampleId;
+	private SampleMaterial sampleMaterial;
+	private String sampleLabName;
+	private String sampleLabExternalId;
+	private String sampleLabPostalCode;
```

Comment:
Facilities will soon have addresses, right?
Does it make sense to only have the postal code and the city? To me it appears plausible to either just have the city (as it can be mapped to Facility.city, or have the full address with postal code, street and house no.

Reply:
If I got this right the information mentioned by you is not delivered by the DEMIS-Message

Reply:
I'll discuss that with @MateStrysewske